### DISCUSSION TITLE
Ampache 6.0.1 Browse Users Uploads missing from side panel

### DISCUSSION THREAD

**Discussion by troycarpenter** [ampache/ampache#3587] [Q&A]
As per the Ampache 6 for admins page , it says there should be a section in Admin->User Tools to allow me to browse users uploads (https://github.com/ampache/ampache/wiki/ampache6-details#admin-users-can-now-browse-all-user-uploads).

I have at least one user who has uploaded files, but I don't see that menu option.  What does it take to get that visible in the menu?

**Comment by troycarpenter** [comment]
To be honest, I don't see any links to Uploads on any screens.  I do have the upload settings, but I don't see where any upload commands show up in the menus.

![image](https://github.com/ampache/ampache/assets/8419000/bd03ff82-d789-49ef-9ee4-4322fd8d91cb)

![image](https://github.com/ampache/ampache/assets/8419000/8c91db8d-3e06-4342-abcb-ed6c16bdb60a)

**Reply by mitchray** [reply]
What happens if you go to /admin/uploads.php

**Reply by troycarpenter** [reply]
That works and that's where I've pointed the users that want to upload since the link doesn't appear in their menu structure for uploads.  We'll see if the fix below takes care of it.

**Comment by lachlan-00** [comment]
code is hiding behind `if (Upload::can_upload(Core::get_global('user'))) {`

which returns a bool checking these
![image](https://github.com/ampache/ampache/assets/1305249/aebf3df0-b66b-4ed5-9b9a-06d87cd39553)

as an admin... you probably shouldn't need this so i should just take that out and use allow_upload instead ofthe user check

**Comment by lachlan-00** [comment]
removed with 8dbafd9c8

**Reply by troycarpenter** [reply]
That seems to have fixed it for the administrator, but I'm not seeing where the user has a link for that.

**Reply by troycarpenter** [reply]
in public/templates/sidebar_home.inc.php
perhaps the setting of the boolean $allow_upload on line 54

**Reply by troycarpenter** [reply]
I short-cutted that boolean by setting it to 1, which got the Uploads menu item to show in Home->Browse.

But am I also just overlooking the actual link to the upload page so users can actually do the upload?

**Reply by troycarpenter** [reply]
Ok, found the same logic in sidebar_preferences.inc.php that needs attention.

**Reply by lachlan-00** [reply]
linking #3589
